### PR TITLE
Added split extensions info

### PR DESCRIPTION
**Pull request by loocool2**
builds on #639, #638, #637, #635, #634

### CODE REVIEW

File: src/help/faq/README.md [hunk 1]
```diff
@@ -308,6 +305,19 @@ Extensions tend to be removed for a variety of reasons:
...
+All of them have been split.
+#### Why do my multisource extension show up as obselete? 
+The multisource extension is obsolete because it has been split. The multisource extension has been removed from the repository. It is not actually obsolete.
+##### How do I move to a split extension?
+Install the extension that your manga is from, uninstall the multisource extension. If the manga shows the source as a bunch of numbers, force close the app. For known issues with split extensions, go to this [GitHub issue](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/5672).
+#### Do I need to migrate? 
+No, unless specifically stated, the internal ID of sources has not changed so migration is not necessary.
+#### What if I uninstall the multisource extension before I install the singular extensions?
+See what extension IDs correspond to what extensions [here](https://pastebin.com/raw/QX2scdRT).
+
```
Comment (Soitora) [inline src/help/faq/README.md:320]: Probably make a newline before each header

File: src/help/faq/README.md [hunk 2]
```diff
@@ -308,6 +305,19 @@ Extensions tend to be removed for a variety of reasons:
...
 A list of extensions that have been removed is available [here](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/3475), it does not include sites that have gone offline.
 
+### Split Extensions
+Extensions have been split due to the sheer number of extensions in a single extension, with Madara reaching over 200 extensions at one point, making it unreasonable for both devs and users to filter through extensions. For a deeper explanation of the issue, [this](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/4287) GitHub issue explains in more detail.
+#### What extensions have been split?
+All of them have been split.
+#### Why do my multisource extension show up as obselete? 
+The multisource extension is obsolete because it has been split. The multisource extension has been removed from the repository. It is not actually obsolete.
+##### How do I move to a split extension?
+Install the extension that your manga is from, uninstall the multisource extension. If the manga shows the source as a bunch of numbers, force close the app. For known issues with split extensions, go to this [GitHub issue](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/5672).
```
Comment (Soitora) [inline src/help/faq/README.md:315]: Probably something like this, it's odd having purely a comma and that text afterwards.

```suggestion
Install the extension that your manga is from and then uninstall the multisource extension. If the manga shows the source as a bunch of numbers, force close the app. For known issues with split extensions, go to this [GitHub issue](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/5672).
```

Maybe some line breaks in this segment as well?

File: src/help/faq/README.md [hunk 3]
```diff
@@ -308,6 +305,19 @@ Extensions tend to be removed for a variety of reasons:
 
 A list of extensions that have been removed is available [here](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/3475), it does not include sites that have gone offline.
 
+### Split Extensions
+Extensions have been split due to the sheer number of extensions in a single extension, with Madara reaching over 200 extensions at one point, making it unreasonable for both devs and users to filter through extensions. For a deeper explanation of the issue, [this](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/4287) GitHub issue explains in more detail.
+#### What extensions have been split?
+All of them have been split.
+#### Why do my multisource extension show up as obselete? 
+The multisource extension is obsolete because it has been split. The multisource extension has been removed from the repository. It is not actually obsolete.
```
Comment (Soitora) [inline src/help/faq/README.md:313]: Do we say multisource or multi-source?

File: src/help/faq/README.md [hunk 4]
```diff
@@ -286,7 +283,7 @@ To solve this issue, follow the below steps.
 
 ### Webtoons
 
-#### How do I fix the `Page list is empty` error?
+#### How do I fix the `Page list is empty` error?
```
Comment (Soitora) [inline src/help/faq/README.md:286]:
```suggestion
#### How do I fix the `Page list is empty` error?
```

**Review by Soitora — COMMENTED** [review]
✍🏻

File: src/help/faq/README.md (hunk 3 above)
Comment (loocool2) [reply src/help/faq/README.md:313]: either one works, dictionary says multisource

File: src/help/faq/README.md (hunk 4 above)
Comment (loocool2) [reply src/help/faq/README.md:286]: deleting this section in the clean up so I'll leave it

**Review by Soitora — APPROVED** [review]

**Pull request merged**